### DISCUSSION TITLE
How to check if an object is off screen?

### DISCUSSION THREAD

**Discussion by Kindlewing** [bevyengine/bevy#5929] [Q&A]
I'm trying to create an asteroids clone in Bevy, and I'm trying to wrap entities when they go off screen.

How do I check this?

**Comment by mockersf** [comment]
You can check with [`Camera::world_to_ndc`](https://docs.rs/bevy/latest/bevy/render/camera/struct.Camera.html#method.world_to_ndc), with the `transform.translation` from your asteroids.
